docs/conf.py

```diff
@@ -0,0 +1,17 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+from rocm_docs import ROCmDocs
+
+docs_core = ROCmDocs("MIOpen Documentation")
+docs_core.run_doxygen()
+docs_core.setup()
+
+os.system("sed -e 's/MIOPEN_EXPORT //g' ../include/miopen/* > .doxygen/")
```

Comment:
We shouldn't modify the source files when generating documentation.

Reply:
[81e377c](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/81e377c569abcba1484353cce47c0018e67462bc)
Instead of changing the source file, this step will make a modified copy because doxygen cannot generate docs properly due to the MIOPEN_EXPORT prefix. The source files will remain the same.

Reply:
Removed in [23e152a](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/23e152af345139b45857c083504ec83490dff615)